src/PBot/Issues/MoveIssue.cs

```diff
@@ -39,6 +40,14 @@ public MoveIssue()
 
             await response.Send(string.Format("Moving issue https://github.com/Particular/{0}/issues/{1}", sourceRepo, issueNumber)).IgnoreWaitContext();
 
+            var client = GitHubClientBuilder.Build();
+            var labels = await client.Issue.Labels.GetForIssue(src.Owner, src.Name, issueNumber);
+            var labelsExist = await LabelUtility.RepositoryHasLabels(dst, labels.ToArray());
+            if (!labelsExist)
+            {
+                await response.Send(string.Format("Could not move issue https://github.com/Particular/{0}/issues/{1} due to missing labels in destination repo {2}. Please execute Pbot.Tests.UpdateRepoLabelsTests test to create label in destination repo.", sourceRepo, issueNumber, targetRepo)).IgnoreWaitContext();
```

Comment:
> Please execute Pbot.Tests.UpdateRepoLabelsTests test to create label in destination repo.

This won't mean anything to non-technical user (and pbot move issue is regularly used by non-technical users). In fact it doesn't even mean much to me :smile: